catalog.html

```diff
@@ -0,0 +1,253 @@
+              <label class="control">
+                <input class="control-input" type="checkbox" name="swimming-pool" checked>
+                Бассейн
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="parking" checked>
+                Парковка
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="wi-fi">
+                Wi-Fi
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Тип жилья:</legend>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="product-group" value="hotel" checked>
+                Гостиница
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="product-group" value="motel">
+                Мотель
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="product-group" value="apartments">
+                Апартаменты
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Стоимсоть, ₽:</legend>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <input class="price-input" type="text" name="price-min" placeholder="от" value="0">
+            </li>
+            <li class="catalog-filter-item">
+              <input class="price-input" type="text" name="price-max" placeholder="до" value="9000">
+            </li>
+          </ul>
+        </fieldset>
+        <button type="submit" name="submit">Применить</button>
+        <button type="reset" name="reset">Сбросить</button>
+      </form>
+    </section>
+
+    <section class="catalog-filter-results">
+      <h2 class="catalog-filter-results-title visually-hidden">Результаты поиска гостиниц</h2>
+      <span class="catalog-filter-results-count">Найдено гостиниц: 38</span>
+      <select class="select-control" name="sorting">
+        <option value="cheap">Сначала дешёвые</option>
+        <option value="expensive">Сначала дорогие</option>
+        <option value="popular">Сначала популярные</option>
+      </select>
+
+      <div class="btn-group">
+        <button class="btn-group-compact visually-hidden" type="button">Комактный вид</button>
```

Comment:
```suggestion
        <button class="btn-group-compact visually-hidden" type="button">Компактный вид</button>
```
Также текст в данных элементах интерфейса нужно обернуть в `span` с классом `visually-hidden`.

Reply:
Похоже, что это замечание вы не заметили. Если видите в гитхабе подобное
![image](https://user-images.githubusercontent.com/17790771/170763858-f2459fb0-1f45-4a78-821e-4dd91c3723de.png)
то это значит, что часть комментариев скрыта за ссылкой.